### ISSUE TITLE
Left side of KH3202R filter seems to be broken

### ISSUE THREAD

**Issue by AaronParsons** [AaronParsons/ugradio#5]
We were putting in a 1 MHz signal mixed with a 1.1 MHz signal into both sides of KH3202 tunable filter on rack in UG lab.  On the left side, we saw no suppression of the 2.1 MHz signal when setting the filter to LPF x10k with the dial set to 100 Hz (nominally, a 1 MHz LPF).  The right side did just fine with switches in the same position.

**Comment by fjlatora** [comment]
I’ve just tested both Channel A and B (CH. A being the left side filter), and compared them to each other.  There should have been a roll off of the signal based on how the unit is functioning.  However, I did notice a couple of problems with channel A.
1.	 Channel A’s output is about %20 less as compared to channel B (as measured on an oscilloscope) with the same input signal and the same cutoff frequency for Both channels.

2.	Channel A seems to be picking up some noise or is being modulated somehow.  It’s a very small amplitude, but it makes the signal trace on the oscilloscope look rather thick.  It's much higher than 60 cycle noise. Because this noise is so small in amplitude, it shouldn't change the actual power delivered to the load at the desired frequency. 


I don’t believe the problem you are describing would be caused by the two problems I have observed, however, I’m not sure of that so I’m working on repairing it now.  It’s a very old piece of equipment and may be difficult to find parts for (assuming its no a generic part like a bad capacitor or ground connection).  If I can’t get this repaired on time, I have some suggestions on an alternative, but would rather discuss this with you in person.
